.github/workflows/conda.yml

```diff
@@ -48,7 +48,6 @@ jobs:
       - name: Upload conda package
         if: |
           github.event_name == 'push'
-          && github.ref == 'refs/heads/main'
```

Comment:
Removing this because the `on` condition already ensures that this will always be true